### PR TITLE
Switch language preferences

### PR DESCRIPTION
**Pull request by rodrigoea**
This PR creates a new tab in User Preferences Modal for users to switch the language.

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,11 @@ build
 dist/
 .rpt2_cache
 .docz/
+.yalc
+
+# YALC (for Erik)
```
Comment (swederik) [inline .gitignore]: .yalc is duplicated. Comment is unnecessary

File: package.json [hunk 2]
```diff
@@ -45,19 +45,26 @@
...
+    "@ohif/i18n": "^0.0.1",
+    "i18next": "^17.0.3",
     "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react-dom": "^16.8.6",
+    "react-i18next": "^10.11.0"
   },
   "dependencies": {
     "@babel/runtime": "7.2.0",
+    "@ohif/i18n": "0.0.1",
```
Comment (swederik) [inline package.json]: Why is is in both peerDependencies AND dependencies?

**Comment by swederik** [comment]
Looks like the deploy preview is failing. Maybe you forgot to check in a file or something? Or possibly it's a filename capitalization issue?

```
Module not found: Can't resolve 'src/__docs__/Wrapper' in '/opt/build/repo/.docz/app'
7:45:57 PM:     at Promise (/opt/build/repo/node_modules/docz-core/dist/index.js:1306:21)
````

File: package.json (hunk 2 above)
Comment (rodrigoea) [reply package.json]: When we’re running react-viewerbase locally and outside of OHIF Viewer, it needs the @ohif/i18n as a dependency to run the docs.

Rollup doesn’t include the package when it’s running in the production version.

**Comment by rodrigolabs** [comment]
@swederik all fixed and deploy preview is now passing successfully!

**Comment by rodrigoea** [comment]
All fixed! Deploy preview is passing successfully!

**Pull request merged**